packages/account/src/Sections/Security/LoginHistory/login-history.jsx

```diff
@@ -20,9 +21,8 @@ const getFormattedData = login_history => {
             /(?<date>[0-9a-zA-Z-]+\s[0-9:]+GMT)[\s](IP=)(?<ip>[\w:.]+)\sIP_COUNTRY=(?<country>([a-zA-Z]{2}))\s(User_AGENT=)(\w.*)(?<name>iPhone|Android)([\W\w]+)\s(?<app>Deriv P2P|Deriv GO)(?<version>[\w\W]+)\s(LANG=)([\w]{2})/
         );
         const date = environment_split[0];
-        const time = environment_split[1].replace('GMT', '');
-        const date_time = convertDateFormat(`${date} ${time}`, 'D-MMMM-YY hh:mm:ss', 'YYYY-MM-DD hh:mm:ss');
-        data[i].date = `${date_time} GMT`;
+        const time = environment_split[1].replace('GMT', ' GMT');
+        data[i].date = `${moment(date).format('YYYY-MM-DD')} ${time}`;
```

Comment:
This change is to fix the translation issue with `Date and time` in login history page.